cart/cart.py

```diff
@@ -1,100 +1,81 @@
 from decimal import Decimal
-
 from django.conf import settings
-
 from shipping.calculator import get_book_shipping_cost
 from store.models import Product
 
-
 class Cart:
```

Comment:
Let's add a unit test to make sure `Cart` instances initialize as expected.

Reply:
Would you like me to add the unit tests to tests.py?

Reply:
Yes, please. I added some comments in the PR about how we can get the mock `request` to contain a `session` property, which should allow the tests to run. After we merge the PR with passing tests, we'll merge this PR with more confidence :-)